Source/NSFileWrapper.m

```diff
@@ -44,325 +44,7 @@
 #import "AppKit/NSImage.h"
 #import "AppKit/NSWorkspace.h"
 
-@implementation NSFileWrapper
```

Comment:
Looks like this whole file is still named wrong.

Reply:
I removed it.